code/modules/clothing/suits/marine_armor/_marine_armor.dm

```diff
@@ -309,7 +307,7 @@
 	name = "\improper M3 pattern general officer armor"
 	desc = "A well-crafted suit of M3 Pattern Armor with a gold shine. It looks very expensive, but shockingly fairly easy to carry and wear."
 	icon_state = "general"
-	armor_bullet = CLOTHING_ARMOR_MEDIUMHIGH
+	armor_bullet = CLOTHING_ARMOR_HIGH
```

Comment:
This doesn't need to change; what gave you the notion this armor is worse than base? It already has 5 more than marine armor; you're just making it have 10 more than marine armor.

Reply:
I presume this is event Armor for a High Command visitor and thought it probably should be on par with the CO. 
I can drop it back down.